### PR TITLE
tests

### PR DESCRIPTION
**Pull request by mlubin**

### CODE REVIEW

**Comment by joaquimg** [comment]
You are thinking about porting tests from MPB or creating new ones from scratch?

File: test/contlinear.jl [hunk 1]
```diff
@@ -35,43 +60,228 @@ function contlineartest(solver::MOI.AbstractSolver, ε=Base.rtoldefault(Float64)
...
+        # Min  x - y
+        # s.t. 100.0 <= x
+        #               y <= 0.0
+        MOI.modifyconstraint!(m, c1, GreaterThan(100.0))
+        MOI.optimize!(m)
+        @test MOI.getattribute(m, MOI.TerminationStatus()) == MOI.Success
+        @test MOI.getattribute(m, MOI.PrimalStatus()) == MPB.FeasiblePoint
+
+        @test MOI.getattribute(m, MOI.ObjectiveValue()) ≈ 100.0 atol=ε
+        @test MOI.getattribute(m, MPB.VariablePrimal(), x) ≈ 100.0 atol=eps
```
Comment (joaquimg) [inline test/contlinear.jl]: Some MPB in the file, I think all of them at `MPB.VariablePrimal`

Comment (joaquimg) [inline test/contlinear.jl]: `MPB.FeasiblePoint` also

**Comment by mlubin** [comment]
@joaquimg, first priority is porting and re-porting (from ``break_everything``)

**Comment by mlubin** [comment]
We have a nontrivial collection of tests now

**Comment by mlubin** [comment]
Some things that are still missing:
- EXP and PSD cones
- Removing constraints
- Adding and removing variables

**Comment by chriscoey** [comment]
two types of PSD cones (scaled and not) and also power cone

**Pull request merged**